content/en/platform/corda/1.6/cenm/resource-requirements.md

```diff
@@ -0,0 +1,20 @@
+---
+aliases:
+- /resource-requirements.html
+date: '2024-10-01'
+menu:
+  cenm-1-6:
+    identifier: cenm-1-6-resource-requirements
+    parent: cenm-1-6-configuration
+    weight: 205
+tags:
+- resource requirements
+- config
+title: CENM VM Sizing Guide
+---
+
+# CENM VM Sizing Guide
+
+When it comes to VM size recommendations for CENM, it's the standard D2 v4 instance for each CENM service:
+* 2 vCPUs
+* 8 GB memory
```

Comment:
Recommended production specification for CENM components running on separate VMs:

- 2 CPU Core, 8 GB Memory